services/handler.py

```diff
@@ -0,0 +1,30 @@
+import json
+from http.server import BaseHTTPRequestHandler
+
+
+class Handler(BaseHTTPRequestHandler):
+
+    def do_HEAD(self):
+        self.send_response(200)
+        self.send_header('Content-type', 'application/json')
+        self.end_headers()
+        
+
+    def do_POST(self):
+        if self.headers.get('Authorization') is None:
+            length = int(self.headers.get('Content-Length'))
+            data = self.rfile.read(length).decode('utf-8')
```

Comment:
As I understand decode('utf-8') allow to get `utf-8` string representation of the binary data.

We need to got JSON so didn't you need to use this ?

```suggestion
            data = json.loads(self.rfile.read(length).decode('utf-8'))
```

Reply:
The Post request sent content type is json so there is no need to add the json.loads() cause that would cause an exception (json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes) as you are trying to load an already json type. However json.dumps in write is needed

Reply:
Ok thanks for the info.